### PR TITLE
OnlineShoppingWebsite

### PR DESCRIPTION
**Pull request by Pragati-3**

### CODE REVIEW

File: pragati/OnlineShopping/product.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+}
+.product-area a:hover .productViewBtn {
+  visibility: visible;
+}
+.popularStores {
+  width: 100%;
+  display: inline-block;
+
+  padding: 15px 0;
+}
```
Comment (shubhali-github) [inline pragati/OnlineShopping/product.css:78]: remove unwanted spaces

File: pragati/OnlineShopping/check.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
+.checkout {
```
Comment (anitapise) [inline pragati/OnlineShopping/check.css:1]: give one tab that means 4 spaces for all css rules

File: pragati/OnlineShopping/contact.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>contact us</title>
+    <link
```
Comment (anitapise) [inline pragati/OnlineShopping/contact.html:7]: tags should be on one line why is it like this

<link
      rel="stylesheet"
      href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"
    />
it should be 
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"/>

File: pragati/OnlineShopping/contact.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        </span>
+      </div>
+    </nav>
+    <div class="container">
+      <h3 class="text-center">Contact us</h3>
+      <br />
+      <div class="row">
+        <div class="col-md-8">
+          <form action="./index.html" method="post">
+            <input
```
Comment (anitapise) [inline pragati/OnlineShopping/contact.html:53]: tags should be in one line dont give space <br /> like this

File: pragati/OnlineShopping/index.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        <div class="offer-card">
+            <img src="./images/images.jpg" width="300px" />
+        </div>
+        <div class="offer-card">
+            <img src="./images/img2.jpg" />
+        </div>
+        <div class="offer-card">
+            <img src="./images/img3.jpg" />
+        </div>
+        <br />
```
Comment (anitapise) [inline pragati/OnlineShopping/index.html:57]: dont give <br /> space and their is no closing tag for br its is only <br>

File: pragati/OnlineShopping/login.css [hunk 6]
```diff
@@ -0,0 +1,76 @@
+html {
+  background: url(./images/notebook.jpg) no-repeat center center fixed;
```
Comment (anitapise) [inline pragati/OnlineShopping/login.css:2]: make it 1 tab means 4 spaces everywhere in the css

File: pragati/OnlineShopping/product.css [hunk 7]
```diff
@@ -0,0 +1,147 @@
+html {
```
Comment (anitapise) [inline pragati/OnlineShopping/product.css:1]: make it 1 tab means 4 spaces everywhere in the css

File: pragati/OnlineShopping/product.css [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+  border-radius: 4px;
+  cursor: pointer;
+}
+.product-area a:hover .productViewBtn {
+  visibility: visible;
+}
+.popularStores {
+  width: 100%;
+  display: inline-block;
+
```
Comment (anitapise) [inline pragati/OnlineShopping/product.css:76]: remove the line here

File: pragati/OnlineShopping/register.css [hunk 9]
```diff
@@ -0,0 +1,76 @@
+html {
+  background: url(./images//images1.jpg) no-repeat center center fixed;
```
Comment (anitapise) [inline pragati/OnlineShopping/register.css:2]: make it 1 tab space for all css

File: pragati/OnlineShopping/style.css [hunk 10]
```diff
@@ -0,0 +1,61 @@
+.nav-link {
+  font-weight: bolder;
```
Comment (anitapise) [inline pragati/OnlineShopping/style.css:2]: make 1 tab space everywere for this css

File: pragati/OnlineShopping/view.css [hunk 11]
```diff
@@ -0,0 +1,136 @@
+.nav-link {
```
Comment (anitapise) [inline pragati/OnlineShopping/view.css:1]: make 1 tab space everywere for this css

File: pragati/OnlineShopping/contact.html [hunk 12]
```diff
@@ -1,103 +1,62 @@
...
+                    <a href="./viewproduct.html">BUY NOW</a>
+                </span>
+            </div>
+        </nav>
+        <div class="container">
+            <h3 class="text-center">Contact us</h3><br />
+            <div class="row">
+                <div class="col-md-8">
+                    <form action="./index.html" method="post">
+                        <input class="form-control" name="name" placeholder="Name..." /><br />
```
Comment (anitapise) [inline pragati/OnlineShopping/contact.html:41]: for br tag no closing required write only <br>
make it everywhere

File: pragati/OnlineShopping/index.html [hunk 13]
```diff
@@ -52,7 +52,7 @@ <h4>
         </div>
         <div class="offer-card">
             <img src="./images/img3.jpg" />
-        </div><br /><br /><br />
+        </div><br ><br ><br >
```
Comment (anitapise) [inline pragati/OnlineShopping/index.html:55]: remove the space make it `<br>`

File: pragati/OnlineShopping/login.html [hunk 14]
```diff
@@ -22,7 +22,7 @@ <h2>
             </div>
             <div class="registerForm">
                 <h3>Login Form</h3>
-                <br />
+                <br >
```
Comment (anitapise) [inline pragati/OnlineShopping/login.html:25]: remove the space make it `<br>`

File: pragati/OnlineShopping/registerform.html [hunk 15]
```diff
@@ -22,7 +22,7 @@ <h2>
             </div>
             <div class="registerForm">
                 <h3>Register Form</h3>
-                <br />
+                <br >
```
Comment (anitapise) [inline pragati/OnlineShopping/registerform.html:25]: remove the space make it `<br>`

File: pragati/OnlineShopping/contact.html [hunk 16]
```diff
@@ -46,9 +46,9 @@ <h3 class="text-center">Contact us</h3><br>
                     </form>
                 </div>
                 <div class="col-md-4">
-                    <b>Customer service:</b> <br>  Phone: +91 9129 209 291<br > E-mail: <a href="mailto:support@myaroma.com">support@aroma.com</a>
+                    <b>Customer service:</b> <br>  Phone: +91 9129 209 291<brs> E-mail: <a href="mailto:support@myaroma.com">support@aroma.com</a>
```
Comment (anitapise) [inline pragati/OnlineShopping/contact.html:49]: what is <brs> please remove and make it <br>

**Pull request merged**